megamek/i18n/megamek/client/messages.properties

```diff
@@ -2090,23 +2150,37 @@ PlanetaryConditionsDialog.labAtmosphere=Atmospheric Pressure
 PlanetaryConditionsDialog.BlowingSands=Blowing Sand
 PlanetaryConditionsDialog.EMI=EMI
 PlanetaryConditionsDialog.labFog=Fog
-PlanetaryConditionsDialog.labGrav=Gravity
+PlanetaryConditionsDialog.labGrav=Gravity (g)
 PlanetaryConditionsDialog.labLight=Light
-PlanetaryConditionsDialog.labTemp=Temperature
+PlanetaryConditionsDialog.labTemp=Temperature (deg C)
```

Comment:
```suggestion
PlanetaryConditionsDialog.labTemp=Temperature (\u00B0C)
```

Reply:
Unicode symbol, which fixes a number of possible issues (had a MekHQ bug that using the icon instead of the unicode led to display issues.

Reply:
Doesnt work on Win10 without setting the font of the label to "Dialog" (which is precisely why I hadnt used it)